cfy_manager/components/prometheus/prometheus.py

```diff
@@ -259,6 +263,22 @@ def _set_selinux_enforcing():
         logger.debug('SELinux is enforced.')
 
 
+def _set_selinux_policies():
+    # Setup Cloudify SELinux policies for Prometheus
+    selinux_state = _get_selinux_state()
+    if selinux_state == 'Disabled':
```

Comment:
Duplicated code (and the disabled comment again), let's maybe move this to utils/selinux.py?